### ISSUE TITLE
Can't find ./bin

### ISSUE THREAD

**Issue by captDaylight** [ethereum/solc-js#143]
I'm seeing this line in the README:

> Note: version strings must match the version substring of the files available in /bin/soljson-*.js

I noticed this because I'm getting an this error: 

```
./~/solc/wrapper.js
Module not found: Can't resolve './bin' in '/node_modules/solc'
```
Is there something missing from the build? I don't have a bin folder in my package.

**Comment by axic** [comment]
That piece of code seems to be slightly outdated. You could use `setupMethods(<loaded file>)` instead.

**Comment by captDaylight** [comment]
Thanks for the quick reply @axic. What would be the "loaded file"? I'm getting this error when I simply try to import the package, so I'm not sure how to get around it. 

Any thoughts on a fix? I could submit a PR.

**Comment by axic** [comment]
I'm not sure why `useVersion` would be called, it is not used anywhere in the project. What do you mean by `import`? Node `require`? ES6 `import`?

**Comment by axic** [comment]
Possibly this is the same issue as #110 ?

**Comment by captDaylight** [comment]
This is happening when all I do is `require` solc into a file. The context I'm compiling with webpack and trying to get solc in the browser.

**Comment by axic** [comment]
Webpack must be doing something weird. I guess it may be calling every function and instrumenting the call graph to see what is being used? Alternatively it does something with it because it has a `require` call.

**Comment by gbalabasquer** [comment]
I'm having the same issue when installing the package to a react project.
`Can't resolve './bin' in '/.../node_modules/solc'`
If I comment the `useVersion` function I get now this problem:
`Module not found: Can't resolve 'module' in '/.../node_modules/require-from-string'`
In this case I comment that module and `loadRemoteVersion` function.

After commenting out those lines, it does not fail anymore at the loading part. However when using `solc.compile` I get this error:
`soljson.cwrap is not a function`. Basically because `require('./soljson.js'))` brings an empty object. I've been reading other threads about these errors but did not find a concrete solution.

**Comment by ch4ot1c** [comment]
When compiling my .ts file (latest Angular, latest solc, trying to use it from within the browser via require('solc')):

ERROR in ./node_modules/require-from-string/index.js
Module not found: Error: Can't resolve 'module' in '/Users/Jon/xx/xx/artist-frontend/node_modules/require-from-string'
resolve 'module' in '/Users/Jon/xx/xx/artist-frontend/node_modules/require-from-string'
  Parsed request is a module
  using description file: /Users/Jon/xx/xx/artist-frontend/node_modules/require-from-string/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/Jon/xx/xx/artist-frontend/node_modules/require-from-string/package.json (relative path: .)
    resolve as module
      /Users/Jon/xx/xx/artist-frontend/node_modules/require-from-string/node_modules doesn't exist or is not a directory
      /Users/Jon/xx/xx/artist-frontend/node_modules/node_modules doesn't exist or is not a directory
      /Users/Jon/xx/xx/node_modules doesn't exist or is not a directory
      /Users/Jon/xx/node_modules doesn't exist or is not a directory
      /Users/Jon/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/Jon/xx/xx/artist-frontend/node_modules
        using description file: /Users/Jon/xx/xx/artist-frontend/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/Jon/xx/xx/artist-frontend/package.json (relative path: ./node_modules)
          using description file: /Users/Jon/xx/xx/artist-frontend/package.json (relative path: ./node_modules/module)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/Jon/xx/xx/artist-frontend/node_modules/module doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/Jon/xx/xx/artist-frontend/node_modules/module.ts doesn't exist
and at the bottom

[/Users/Jon/xx/xx/artist-frontend/src/module.ts]
[/Users/Jon/xx/xx/artist-frontend/src/module.js]
[/Users/Jon/xx/xx/artist-frontend/src/module]
 @ ./node_modules/require-from-string/index.js 3:13-30
 @ ./node_modules/solc/wrapper.js
 @ ./node_modules/solc/index.js
 @ ./src/app/app.component.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://localhost:4200 ./src/main.ts
Going to go ahead and use a Browserify option and see how that goes... But I hope this can help! require-from-string and Module inside it are the culprits from my stack trace.

**Comment by ch4ot1c** [comment]
**Solved my issue!**

`require-from-string` (which is a dependency) was freaking out because of this commit:
https://github.com/floatdrop/require-from-string/commit/80783b2ca46daa4157241503e434f10a24bc4c67

Switched out ^ that line (or really, `npm install require-from-string@1.2.0 --save`), and voila!

I suggest we pin `require-from-string` to v1.2.0, due to this 'bug' on their end. v1.2.1 automatically installs when `npm install`ing the latest `solc-js`, which is the incompatible version.

Making a PR.

**Comment by frewinchristopher** [comment]
I'm trying `var solc = require('solc')` within a `create-react-app`, and also getting this error My `package.json` has `"require-from-string": "1.2.0"` but I'm still getting this error.

doing 

`node`
`> var solc = require('solc')`

in the same project works fine though, and on this command line `solc` works fine as expected

**Comment by axic** [comment]
Apparently fixed by #261 in late 2018.
